### PR TITLE
CLC-5401 Style tabs in webcast widget per Instructure guide

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-5401

### CODE REVIEW

File: src/assets/templates/widgets/webcast.html [hunk 1]
```diff
@@ -3,25 +3,18 @@
...
-          </li>
-        </ul>
-      </div>
+    <div data-ng-if="eligibleForSignUp && eligibleForSignUp.length > 0" class="bc-tab-navigation">
+      <ul data-ng-if="videos || audio" class="bc-tab-group cc-clearfix-container" role="tablist">
+        <li class="bc-tab"
+            data-ng-class="{'bc-tab-active':(currentTabSelection === tabOption)}"
+            data-ng-repeat="tabOption in outerTabOptions">
+          <a data-ng-click="switchTabOption(tabOption)"
+            aria-selected="$first"
```
Comment (christianv) [inline src/assets/templates/widgets/webcast.html]: @johncrossman we probably need something similar as the `active` class: `data-ng-class="{'bc-tab-active':(currentTabSelection === tabOption)}"`

File: src/assets/templates/widgets/webcast.html [hunk 2]
```diff
@@ -66,25 +59,18 @@ <h1 class="cc-visuallyhidden" data-ng-if="isEmbedded">Webcast Sign-up</h1>
...
-            </li>
-          </ul>
-        </div>
+      <div data-ng-if="videos || audio" class="bc-tab-navigation">
+        <ul class="bc-tab-group cc-clearfix-container" data-ng-if="videos.length && audio.length" role="tablist">
+          <li class="bc-tab"
+              data-ng-class="{'bc-tab-active':(currentSelection === selectOption)}"
+              data-ng-repeat="selectOption in selectOptions">
+            <a data-ng-click="switchSelectedOption(selectOption)"
+              aria-selected="{{currentSelection === selectOption}}"
```
Comment (christianv) [inline src/assets/templates/widgets/webcast.html]: @johncrossman we should probably use `ng-attr` here as well.

File: src/assets/stylesheets/_canvas_base.scss [hunk 3]
```diff
@@ -263,6 +263,48 @@ button.bc-button-link {
...
+    text-decoration: none;
+
+    .bc-tab {
+      background: transparent;
+      border: 1px solid transparent;
+      border-bottom: 0;
+      border-radius: 3px 3px 0 0;
+      float: left;
+      list-style: none;
+      margin: 0 0.2em 1px 0;
```
Comment (christianv) [inline src/assets/stylesheets/_canvas_base.scss]: @paulkerschen `scss-lint` might trip over this, we probably should use `.2em` instead.

**Comment by paulkerschen** [comment]
Thanks for comments @christianv - Travis is chewing on amendment.

**Comment by johncrossman** [comment]
thanks @paulkerschen

**Pull request merged**